src/api/test/functional/attributes_test.rb

```diff
@@ -131,6 +126,10 @@ def test_create_type
     assert_response :success
     get "/attribute/TEST/Dummy/_meta"
     assert_response :success
+    login_Iggy
+    delete "/attribute/TEST/Dummy/_meta"
+    assert_response 403
+    login_adrian
```

Comment:
@adrianschroeter you can validate API requests and responses in rspec as long as call you call `render_views` in the spec (this was mentioned in the issue I created).

@mschnitzer ok thats fine